mkt/users/models.py

```diff
@@ -83,6 +82,9 @@ class UserProfile(OnChangeMixin, ModelBase, AbstractBaseUser):
                             editable=False)
     enable_recommendations = models.BooleanField(default=True)
 
+    shown_dev_agreement = models.DateTimeField(null=True, blank=True)
+    read_dev_agreement = models.DateTimeField(null=True, blank=True)
```

Comment:
That's what I'd prefer, but I didn't want to change wording with the existing field, which is `read` :/